libbeat/kibana/testdata/fields.yml

```diff
@@ -1,5 +1,8 @@
 - key: test
   title: Test fields.yml
+  source_filters: 
```

Comment:
Can we name this `kibana.source_filters` as I think this is Kibana only? We discussed in the past to introduce a `kibana` specific namespace for the kibana only fields options. Lets start with the new additions for now and migrate the old ones step by step.